src/services/bundle-descriptor-converter-service.ts

```diff
@@ -28,17 +28,33 @@ const WIDGET_DESCRIPTOR_VERSION = 'v2'
 const BUNDLE_DESCRIPTOR_VERSION = 'v2'
 const BUNDLE_DESCRIPTOR_NAME = 'descriptor' + DESCRIPTOR_EXTENSION
 
+export enum ThumbnailStatusMessage {
+  NONE = '',
+  FILESIZE_EXCEED = 'Thumbnail filesize is too big. Make sure the thumbnail filesize is 100 KB or below.',
```

Comment:
Bundle thumbnail file ignored because the file size exceeds 100 KB

Reply:
done